ISSUE_TEMPLATE.md

```diff
@@ -1,12 +1,20 @@
 #### Affected Version
 <!-- Please ensure you are using the latest yay-git package -->
 <!-- Use `yay -V` to get installed version -->
-<!-- Example: `yay v3.373` -->
+<!-- Example: `yay v8.1139.r0.g9ac4ab6 - libalpm v11.0.1` -->
 
 #### Issue
 
+
+<!-- The following sections may be left out if not relevant -->
 #### Steps to reproduce
-<!-- Exact commands are useful -->
+<!-- Use exact commands where applicable -->
 1.
 2.
 3.
+
+#### Output
+<!-- Include the FULL output -->
+<!-- Include any relevant commands/configs -->
```

Comment:
Maybe adding an

```
if relevant include the output of yay -Pg
```
would help people discover this amazing flag

Reply:
Noted, leaving the commands/configs in because it's not just the yay config that me be relevant.